docs/community/developer_call_notes/2020_05_07.rst

```diff
@@ -29,6 +29,7 @@ Organizational updates
       wording matter
     - currently many core members are from Microsoft, but the project is an
       independent open source project
+
```

Comment:
Did you mean to do this?

Reply:
Yes, this was a warning on the ReST because you need empty lines before a list (and this is a list, even though it's nested)